src/app/utils/quiz.js

```diff
@@ -0,0 +1,79 @@
+let baseApi;
+let questionScope = [];
+let type = '';
+const question = {};
+
+async function fetchItems(api) {
+  await fetch(api)
+    .then((item) => item.json())
+    .then((item) => {
+      questionScope.push(...item.results);
+      if (item.next !== null) {
+        fetchItems(item.next);
+      }
+    });
+}
+
```

Comment:
This empty line is unnecessary.

Reply:
Empty line should be in the end of every file. It is properly formatted.